src/store/index.ts

```diff
@@ -1,10 +1,19 @@
-import { configureStore } from '@reduxjs/toolkit';
+import { combineReducers, configureStore } from '@reduxjs/toolkit';
 import { appReducer } from './app/reducer';
+import { launchReducer } from '../components/LaunchFilter/reducers/launchReducer';
+
+const reducer = combineReducers({
+  app: appReducer,
+  launches: launchReducer,
+});
 
 export const store = configureStore({
-  reducer: {
-    app: appReducer,
-  },
+  reducer,
+  middleware: getDefaultMiddleware =>
+    getDefaultMiddleware({
+      serializableCheck: false,
+    }),
```

Comment:
why do you add this middleware?

Reply:
To avoid the error "A non-serializable value was detected in the state"  then you use @reduxjs/toolkit. [Serializability Middleware](https://redux-toolkit.js.org/api/serializabilityMiddleware)
This error was

Reply:
To avoid the error "A non-serializable value was detected in the state"  using `redux-toolkit` [Serializability Middleware](https://redux-toolkit.js.org/api/serializabilityMiddleware)